client/tests/integration/roles.rs

```diff
@@ -160,25 +159,32 @@ fn unregistered_role_removed_from_account() -> Result<()> {
 
     let role_id: <Role as Identifiable>::Id = "root".parse().expect("Valid");
     let alice_id: <Account as Identifiable>::Id = "alice@wonderland".parse().expect("Valid");
+    let bob_id: <Account as Identifiable>::Id = "bob@wonderland".parse().expect("Valid");
+
+    // Registering Bob
+    let register_bob = RegisterBox::new(Account::new(bob_id.clone(), []));
+    test_client.submit_blocking(register_bob)?;
 
     // Register root role
-    let register_role = RegisterBox::new(NewRole::new(role_id.clone()).build());
+    let register_role = RegisterBox::new(
+        Role::new(role_id.clone()).add_permission(CanSetKeyValueInUserMetadata::new(alice_id)),
+    );
     test_client.submit_blocking(register_role)?;
 
     // Grant root role to Alice
```

Comment:
```suggestion
    // Grant root role to Bob
```
Same for the rest of this test